src/utils/stringUtils.js

```diff
@@ -38,6 +38,8 @@ export const cleanString = {
     }
 };
 
+// Removed from Organisation.vue, but I've not deleted it in case it is needed again.
```

Comment:
No need to comment this function, since it is removed from the Organisation.vue, it won't impact anything.

Reply:
Are you suggesting that the function simply be deleted?

Reply:
No, it should remain like it is. Not to comment that function.